docs/CodingStyleGuidelines.md

```diff
@@ -140,8 +140,8 @@ foo_drv.sv:
 | `_c`    | Classes                     | `_t`     | Typedefs, Misc. Types |
 | `_e`    | Enumerated Types            | `_s`     | Structs               |
 | `_pkg`  | Packages                    | `_cb`    | Clocking Blocks       |
-| `_intf` | Interface  Types            | `_mp`    | Modports              |
-| `_i`    | Interface Instances         | `_cnstr` | Constraints           |
+| `_if`   | Interface  Types            | `_mp`    | Modports              |
+| `if_i`  | Interface Instances         | `_cnstr` | Constraints           |
```

Comment:
I see that we have not been consistent in the way we have named our interface instances. Some interface instances have the postfix _i (actually if_i), and some have _if. As the guidelines originally state that we sould use _i, I changed this to _if_i, and not only _if. To be consistent we will need some change in the code base, but should we go for _if_i, _if or only _i?

Reply:
I would prefer `_if`.  Typically you will see `_i` to indicate an input signal to a module, so using it to designate an interface may confuse some people.

Reply:
Seems we use `..._i` for module instances. Sometimes also `u_...`.

Seems we use `..._if` for both interface types and interface instances.